source/releasenotes/2024-12-10-drupal-7-d7es.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Tag1 Extended Support module installed on Drupal 7 upstream
```

Comment:
In an internal draft we had the title here as 

```
Drupal 7 Long-Term Support now operational on Pantheon
```

I think it is wise to immediately say in the prose of the release note that the specific observable change is the inclusion of the module. But to my knowledge there are a few more internal things we've done (that don't necessarily need to be spelled out in the release note) that add up "the whole idea is operational"

Reply:
```suggestion
title: Drupal 7 Long-Term Support now operational on Pantheon
```